devops/README.md

```diff
@@ -20,34 +20,47 @@ Read more about Loki [here](https://grafana.com/oss/loki/).
 
 The current proxy implementation is Caddy. Read more about Caddy [here](https://caddyserver.com/).
 
+3. Minio - a high-performance object storage server
+4. Memcached - a high-performance, distributed memory object caching system
+
 ### Deployment
 
 Deploy the Loki Logstack using the following command:
 
 ```bash
-helm template bcwallet ./devops/charts/loki-logstack \
---set-string namespace=abc123-dev \
---set-string proxyUserName=<USER_NAME> \
---set-string proxyPassword=<PASSWORD> | \
-oc apply -n e79518-dev -f -
+helm template bcwallet ./devops/charts/loki-logstack -f ./devops/charts/loki-logstack/values_test.yaml \
+--set-string namespace=ca7123-test \
+--set-string minio_access_key=$MINIO_ACCESS_KEY \
+--set-string minio_secret_key=$MINIO_SECRET_KEY \
+--set-string proxyUserName=$PROXY_USER_NAME \
+--set-string proxyPassword=$(htpasswd -nbB $PROXY_USER_NAME $PROXY_PASSWORD| awk -F: '{ print $2 }'|tr -d '[:space:]'|base64)| \
+oc apply -n ca7123-test -f -
 ```
 
 The parameters passed in via the `--set-string` argument for this command are as follows:
 
-| Value         | Description                                                                                         |
-| ------------- | --------------------------------------------------------------------------------------------------- |
-| namespace     | The namespace in which to deploy the Loki Logstack. This is used by Caddy to find the Loki service. |
-| proxyUserName | The username for the Loki Proxy. This is part of the authentication credentials.                    |
-| proxyPassword | The password for the Loki Proxy.This is part of the authentication credentials.                     |
+| Value            | Description                                                                                         |
+| ---------------- | --------------------------------------------------------------------------------------------------- |
+| namespace        | The namespace in which to deploy the Loki Logstack. This is used by Caddy to find the Loki service. |
+| proxyUserName    | The username for the Loki Proxy. This is part of the authentication credentials.                    |
+| proxyPassword    | The password for the Loki Proxy.This is part of the authentication credentials.                     |
+| minio_access_key | The access key associated with Minio                                                                |
+| minio_secret_key | The secret key associated with Minio                                                                |
 
-Once deployed there will be two pods running that can be verified with the following command:
+Once deployed there will be several pods running, depending on your replication count, that can be verified with the following command:
 
 ```console
-➜  vc-wallet-mobile ✗ oc get pods -l "app.kubernetes.io/name=logstack"
+➜  vc-wallet-mobile git:(feat/loki-ha) ✗ oc get pods -l "app.kubernetes.io/name=logstack"
```

Comment:
this is v nitpicky but there are a few instances of vc-wallet-mobile rather than bc-wallet-mobile